library/keystone_group.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python
```

Comment:
Tried it briefly, didnt work and had to do a bunch of others anyway, so just plowed ahead.